examples/verifying_gfd/verifying_gfd2.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import desbordante
+
+class bcolors:
+    HEADER = '\033[95m'
+    WARNING = '\033[93m'
+    ENDC = '\033[0m'
+
+graph_name = 'blogs_graph'
+modified_graph_name = 'two_authors_blogs_graph'
+gfd_name = 'one_author_gfd'
+
+GRAPH = Path(f'examples/verifying_gfd/datasets/graphs/{graph_name}.dot')
+MODIFIED_GRAPH = Path(f'examples/verifying_gfd/datasets/graphs/{modified_graph_name}.dot')
+GFD = Path(f'examples/verifying_gfd/datasets/gfds/{gfd_name}.dot')
+
+GRAPH_IMAGE = Path(f'examples/verifying_gfd/figures/graphs/{graph_name}.png')
+MODIFIED_GRAPH_IMAGE = Path(f'examples/verifying_gfd/figures/graphs/{modified_graph_name}.png')
+GFD_IMAGE = Path(f'examples/verifying_gfd/figures/gfds/{gfd_name}.png')
```

Comment:
Let's define `GRAPHS_DATASETS_FOLDER_PATH` and use it here.
See `anomaly_detection.py` as example